README.md

```diff
@@ -201,6 +201,23 @@ Tags can also be set globally using setTags
 client.setTags(['Tag1', 'Tag2']);
 ```
 
+### Timestamp
```

Comment:
It would be  good to add why this could be use or when this could be useful (e.g. the Issue mentioned with the usage of `winston`